Templating/Helper/ImagineHelper.php

```diff
@@ -40,7 +43,13 @@ public function __construct(CacheManager $cacheManager)
      */
     public function filter($path, $filter, array $runtimeConfig = array())
     {
-        return $this->cacheManager->getBrowserPath($path, $filter, $runtimeConfig);
+        $pathParts = explode('?', $path, 2);
```

Comment:
you can try this, to keep list function use.

`       
 list($path, $queryString) = array_pad(explode('?', $path, 2), 2, null);
`